modules/oxide/src/less/theme/components/context-toolbar/context-toolbar.less

```
@@ -8,11 +8,15 @@
     // Enable automatic flip when toolbar would overflow viewport, very new, not recongnized by stylelint yet
     // stylelint-disable-next-line property-no-unknown -- CSS Anchor Positioning spec (Chrome 125+)
-    position-try-fallbacks: flip-block, flip-inline, flip-block flip-inline;
+    position-try-fallbacks:
+      flip-block,
+      flip-inline,
+      flip-block flip-inline;
     z-index: @z-index-menu;
 
-    background-color: #ffffff;
-    border: 1px solid #e0e0e0;
+    background-color: var(--tox-private-background-color, @background-color);
```

Stylelint (17.4.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)